synapse/config/server.py

```diff
@@ -207,6 +207,7 @@ class HttpListenerConfig:
     additional_resources: Dict[str, dict] = attr.Factory(dict)
     tag: Optional[str] = None
     request_id_header: Optional[str] = None
+    experimental_cors_msc3886: bool = False
```

Comment:
please can you pop a comment on describing what this means? I know it's a bit hypocritical since none of the other attributes do, but blame that on them being ancient :)

Reply:
Done.